TravelPoints-UI/src/index.html

```diff
@@ -10,6 +10,11 @@
         integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <script>
+    if (global === undefined) {
+      var global = window;
+    }
+  </script>
```

Comment:
why is this needed?

Reply:
I received an error if not adding it:
Uncaught ReferenceError: global is not defined